code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -12,6 +12,8 @@
 			msg += "This creature is impregnated and [reagents.get_reagent_amount(/datum/reagent/toxin/xeno_growthtoxin) > 0 ? "" : "not"] inoculated with Larval Accelerant. \n"
 		else if(chestburst == 2)
 			msg += "A larva escaped from this creature.\n"
+		else if (headbitten == TRUE)
+			msg += "This creature has been purged of vital organs in the head.\n"
```

Comment:
```suggestion
		else if (headbitten)
			msg += "This creature has been purged of vital organs in the head.\n"
```
Additionally, are you sure about this being an elseif, as chestbursts and headbites might not be mutally exclusive

Reply:
The idea was to make them mutally exclusive, yeah. Still, it wouldn't hurt to have both as an option.